typespec-tests/src/main/java/com/cadl/armstreamstyleserialization/fluent/models/SalmonInner.java

```diff
@@ -75,6 +75,7 @@ public String kind() {
      * 
      * @return the friends value.
      */
+    @Override
```

Comment:
Seems not correct. These properties are defined in `SalmonInner` itself:
https://github.com/Azure/autorest.java/blob/bf0facc6325a2b5fcfffddd816a7c98328aab395/typespec-tests/tsp/arm-stream-style-serialization.tsp#L52-L57